### PR TITLE
VSP-653 [iOS] Landing Page - My Public Galleries

### PR DESCRIPTION
**Pull request by luciancerbu-vsp**
- Public Galleries List will show up before searching. 
- Empty galleries (no published files / materials) should not show up in the public galleries list
- Added UI elements for being able to navigate to user public archives.
- Added permissions for viewing of profile page.

### CODE REVIEW

File: Permanent/ViewControllers/Profile/PublicProfilePageViewController.swift [hunk 1]
```diff
@@ -249,51 +217,59 @@ extension PublicProfilePageViewController: UICollectionViewDataSource {
             let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ProfilePageAboutCollectionViewCell.identifier, for: indexPath) as! ProfilePageAboutCollectionViewCell
             let shortDescriptionValue = viewModel?.blurbProfileItem?.shortDescription
             
-            cell.configure(shortDescriptionValue ?? viewModel?.archiveType.shortDescriptionHint)
+            if isEditDataEnabled {
```
Comment (v-rusu) [inline Permanent/ViewControllers/Profile/PublicProfilePageViewController.swift:220]: is this still necessary?

File: Permanent/ViewControllers/SideMenuViewController.swift [hunk 2]
```diff
@@ -230,6 +233,13 @@ extension SideMenuViewController: UITableViewDataSource, UITableViewDelegate {
             newRootVC.archiveData = archive
             AppDelegate.shared.rootViewController.changeDrawerRoot(viewController: newRootVC)
             
+        case .publicGallery:
+            guard let archive = viewModel?.getCurrentArchive() else { return }
+            let newRootVC = UIViewController.create(withIdentifier: .publicGallery, from: .main) as! PublicGalleryViewController
+            newRootVC.archive = archive
+            newRootVC.viewModel?.account
```
Comment (v-rusu) [inline Permanent/ViewControllers/SideMenuViewController.swift:240]: I don't think the line above does anything

File: Permanent/ViewControllers/SideMenuViewController.swift [hunk 3]
```diff
@@ -166,9 +169,9 @@ extension SideMenuViewController: UITableViewDataSource, UITableViewDelegate {
     }
     
     func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
-        if [LeftDrawerSection.publicGallery.rawValue, LeftDrawerSection.header.rawValue, LeftDrawerSection.manage.rawValue].contains(section) {
-            return nil
-        }
+//        if [LeftDrawerSection.publicGallery.rawValue, LeftDrawerSection.header.rawValue, LeftDrawerSection.manage.rawValue].contains(section) {
```
Comment (v-rusu) [inline Permanent/ViewControllers/SideMenuViewController.swift:172]: please delete commented code

File: Permanent/ViewControllers/Profile/PublicProfilePageViewController.swift (hunk 1 above)
Comment (luciancerbu-vsp) [reply Permanent/ViewControllers/Profile/PublicProfilePageViewController.swift]: I removed it, it's not necessary anymore.

File: Permanent/ViewControllers/SideMenuViewController.swift (hunk 2 above)
Comment (luciancerbu-vsp) [reply Permanent/ViewControllers/SideMenuViewController.swift]: newRootVC.archive is not used anymore.

File: Permanent/ViewControllers/SideMenuViewController.swift (hunk 3 above)
Comment (luciancerbu-vsp) [reply Permanent/ViewControllers/SideMenuViewController.swift]: I will do that

**Pull request merged**